### ISSUE TITLE
is it possible to have more documentss?

### ISSUE THREAD

**Issue by carlhung** [pmacro/Suit#8]
I tried to use. but still don't know how to use.
I only know UIKit.

**Comment by pmacro** [comment]
Yes, definitely!  This is important and at the top of my mind.  I'll try to pull together at least something basic today to start with.

**Comment by carlhung** [comment]
thx

**Comment by pmacro** [comment]
There'll be a little pause on this.  With the new Swift technologies, such as FunctionBuilder, we need to take some time to research, especially when RxSwift was always intended to be part of Suit (hence the work-in-progress RxSwift integration under Stride).  It may be that we start moving in a direction more like SwiftUI.  But we need to investigate.  And this is open to all interested participants to share ideas!

RxSwift + a DSL for UI declaration could be interesting.  I'll probably push a branch with some experiments in this area.
